app/api/etc/types.js

```diff
@@ -1,14 +1,44 @@
+// @flow
+import BigNumber from 'bignumber.js';
+
+export type EtcTransactioPassphrase = string;
```

Comment:
There is a typo in `EtcTransactioPassphrase` 😉

Reply:
good eye 😉